### ISSUE TITLE
T-Shirt Designs and Printing

### ISSUE THREAD

**Issue by Merelleya** [HaxeFoundation/Project-Management#33]
I am opening a new issue for this, since it is not exactly what I asked @markknol to do in #26.

If we are to have T-shirts, we need to sort out the following:
- [ ] Design and color options
- [ ] Printing service
- [ ] Shipping

We need to decide, whether we want people to order the shirts through the Haxe Foundation/Site, or whether they should order them directly at the printers. If we have people order through the foundation, we need to put some thought into the workflow and/or find a printer that sort of "integrates" with us.

If we have people order them at the printers themselves, we could provide the designs and just let people choose where to order or we could set up something custom with a printing service. Either way, the fact that we are all over the world could be a problem. We should at least look for one solution in Europe and one in the US.

The same goes for the other printed goods.

Does anybody feel like designing t-shirts today?
- [ ] T-Shirts
- [ ] Coffee Mugs (?)
- [ ] Buttons

**Comment by delahee** [comment]
We used comboutique.com back in the days. 
I would vote for a solution that would create a money income for the foundation, as creating custom shirts is really mainstream these days.

**Comment by mikicho** [comment]
I want two T-shirts and 2 coffee mugs . Plz :D
Not sure about the mugs but maybe this site can help with the shirts http://www.spreadshirt.com

**Comment by mikicho** [comment]
I mean this:http://www.spreadshirt.com/-C59

**Comment by fullofcaffeine** [comment]
By the way, GameDuell made a really cool design for a Haxe t-shirt that the gameduell guys were wearing at the WWX2015. I asked them if they could release the design under (maybe) a creative commons license, let's see! /cc @catdawg

**Comment by catdawg** [comment]
I think that when we had the "tech talk" here with Nicolas, there was some kind of deal that the tshirt design was to be shared. I finally have the source image file.

Here it is: https://drive.google.com/folderview?id=0B_KzBG-j-HgufnVTUFE0Z3ZVMVFOVW9BOGlLcWswZFJGU3V2Z09zdWJnRTdsMlVXdFU4NVE&usp=sharing

Have fun with it!

**Comment by ConfidantCommunications** [comment]
Are you open to more design submissions?

**Comment by Merelleya** [comment]
yes, we are^^

Am 22/06/2015 um 16:32 schrieb Allan Dowdeswell:

**Comment by ConfidantCommunications** [comment]
Okay here's an idea. https://drive.google.com/folderview?id=0B6UVeFM3eTGBfmJmZGZBalB3QWFMRzllYlU3YVpfbnYwMEt0b1FDb1hUeWpneldOM2xjRFE&usp=sharing

I would need to set up the inks for screen printing if approved.

**Comment by ConfidantCommunications** [comment]
Left speechless, eh? ;)

**Comment by MatthijsKamstra** [comment]
This issues doesn't seem to hard to fix, black tee with yellow logo (@markknol made the branding guide for it).
And it seems that OpenFL already figured it out : http://www.amazon.com/OpenFL-Logo-T-Shirt/dp/B016ARC74I

And provided a Haxe tee: http://www.amazon.com/OpenFL-Haxe-Logo-Vertical-T-Shirt/dp/B016APSU5A/ref=pd_srecs_cs_193_1?ie=UTF8&refRID=11PF67ESH2MDR2B9Y8X7&dpID=41Q726prrYL&dpSrc=sims&preST=_SL500_AC_SR170%2C250_

(although I personally think the haxe logo is too big)

So time to add it to the website?
And close this issue?

Btw I am willing to use my awesome design powers to design a mug, so we only need to find a company how is willing to do our dirty works: http://www.cafepress.co.uk/
They have mugs and tees (and more) 
http://www.cafepress.com/cp/info/help/pricing_policy.aspx
All we need is an account (done by HF)

**Comment by markknol** [comment]
Aiii, that openFL Haxe logo shirt.. @Merelleya Isn't it weird the openFL team sells shirts with the Haxe logo? 

To be honest, I think goodies like shirts/mugs can be a good thing, but only if they are done properly (aka not ugly or as the branding guide says _elaborated with gold jewelry_). It's a real waste to produce shirts that no one wares (or only once at events). The color and proportions of the logo should be good and the shirt of fine quality. 

@ConfidantCommunications I think the shirt you've designed is not in line with how the Haxe branding should be represented. 
@MatthijsKamstra  If you want to pull this off, please do. I'd love to do a design check on it.

**Comment by ibilon** [comment]
@markknol they do? I don't see any http://www.amazon.com/s/ref=w_bl_sl_s_ap_web_7141123011?ie=UTF8&node=7141123011&field-brandtextbin=OpenFL

EDIT: found the one I think you're talking about (http://www.amazon.com/OpenFL-Haxe-Logo-T-Shirt/dp/B016APPWNS/ref=pd_d0_recs_v2_cwb_ap_6?ie=UTF8&refRID=1ACT4C49E4JTT49G1BWQ&dpID=41X3deege7L&dpSrc=sims&preST=_SL500_SR160%2C160_), strange it's not on the main list.

**Comment by larsiusprime** [comment]
@ibilon The ones not on the main list are probably not intended to be public, there's mario and link and stuff too that probably couldn't ever be released publicly, so everything except the OpenFL shirts on the main list are most likely just for personal orders and the like, or the result of messing around with the interface.

**Comment by jgranick** [comment]
Hi everyone,

The shirts are good quality. American Apparel have always been my favorite T-shirts, of all the different printed shirts I received at different events.

There are two OpenFL shirts, and one Lime shirt:

http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias%3Daps&field-keywords=openfl+shirt

Yes, I was testing three different Haxe shirt designs, but none are live (and should not have publicly available URLs). The sometimes complex Haxe politics would dictate that helping produce T-shirts would lead to trouble :frowning:

**Comment by Merelleya** [comment]
As usual the brand discussion is dependent on design input. Since we don't have a resident designer, @markknol I am naming you Brand Guardian for now.

**Comment by MatthijsKamstra** [comment]
Bam lets get this discussion going:

only two designs, but completely in line with the brand-guides (checking with newly appointed Brand Guardian @markknol)
and possible with just one side printed

**version 0**
![haxe_tshirt_v01_white tee 0](https://cloud.githubusercontent.com/assets/907219/10527838/fdca73d6-7392-11e5-833b-d9d4e2a60939.png)
![haxe_tshirt_v01_black tee 0](https://cloud.githubusercontent.com/assets/907219/10527837/fdc6026a-7392-11e5-9ace-7daa9d415d83.png)

**version 1**
![haxe_tshirt_v01_white tee 1](https://cloud.githubusercontent.com/assets/907219/10527843/05ebe5c2-7393-11e5-90fe-62720d888237.png)
![haxe_tshirt_v01_black tee 1](https://cloud.githubusercontent.com/assets/907219/10527842/05e738c4-7393-11e5-82ff-cc5620a7a96d.png)

**Comment by MatthijsKamstra** [comment]
Bam mug
![haxe_mug_v01](https://cloud.githubusercontent.com/assets/907219/10528201/0ed1211e-7395-11e5-82b6-035576a8ac28.png)

on both sides the Haxe logo

**Comment by larsiusprime** [comment]
I like them all! Only comment is that the shirts with the small logo on the side looks a bit large for a pocket-sized logo? Is there a standard guideline for that or are my eyes just off?

Everything else looks super duper swell.

**Comment by MatthijsKamstra** [comment]
if you look at the black, the white doesn't look that off.
I still have to consult my wife, she is in fashion, about size and positioning of the logo.
But I wanted to have something to discuss (my favourite is version 1 with haxe across the chest )

**Comment by mikicho** [comment]
I agree with @larsiusprime, I created 2 months ago a Haxe shirt and the pocket logo should be about 10-12 cm.
Also, I recommend that you write in both sides the official website url.
I like that mug!

**Comment by Merelleya** [comment]
This is looking good. I hope we can settle on a design in the next two weeks so that we can look into setting this up with t-shirt printing services. We should probably get one in Europe and one in the US.

**Comment by markknol** [comment]
They all look very good. Indeed the small ones looks a bit big, but the colors and backgrounds are good. 
Can we also have a black mug?

**Comment by MatthijsKamstra** [comment]
@markknol sure, I used a vector file that wasn't easy to change. But you probably get the idea. 

@Merelleya I wanted to use something like http://www.cafepress.co.uk/ , they handle all the shipping, etc. You can set you profit margin, can choose from a wide range of products and they have "stores" in different countries.

@mikicho I don't want to add an url (to the t-shirt) because I hope when you wear this tee you have a conversation starter. And I think that Haxe users can explain more passionate why to use Haxe then the website ever could.

**Comment by waneck** [comment]
I think it would be nice if we had some t-shirts that show a little what
Haxe is about.
Something like all the logos of what Haxe targets, and "CAN YOUR
PROGRAMMING LANGUAGE TARGET ALL THIS?" (Or I bet your programming language
can't target all this), with the Haxe logo on the back

Also some slogans like "MACRO ALL THE THINGS".

**Comment by waneck** [comment]
I'd love to see a pixelized Haxe logo as well

**Comment by ConfidantCommunications** [comment]
On the ones with logo on pocket, I would suggest having a smaller logo, like 1/2 as tall.

**Comment by MatthijsKamstra** [comment]
the quick and dirty black mug version
![haxe_mug_v02_black](https://cloud.githubusercontent.com/assets/907219/10601379/8a462432-7711-11e5-9b9a-5c970dae937b.png)

@markknol do you know which font is used in the Haxe logo? I guess it's "Helvetica Neue Black" (there are others that look similar, but I think that Helvetica is the best guess)

@waneck that is always possible, but I first would like the cover the "basics". I would not be convertible wearing "MACRO" shirt because I don't use them. I would wear a tee with "All your targets are belong to us" :stuck_out_tongue_winking_eye: 
The pixel logo is difficult: I am not a pixel artist and the logo doesn't degrade nicely when I automate it in Photoshop

![pixalate](https://cloud.githubusercontent.com/assets/907219/10602847/15a14242-771b-11e5-8f27-c4befa1c48e2.gif)

**Comment by MatthijsKamstra** [comment]
![haxe_tshirt_v01_black tee 0](https://cloud.githubusercontent.com/assets/907219/10603028/3ed8abfe-771c-11e5-89cf-3c1311da244e.png)
![haxe_tshirt_v01_black tee 1](https://cloud.githubusercontent.com/assets/907219/10603026/3ed71fbe-771c-11e5-9bc8-25502851b9c2.png)
![haxe_tshirt_v01_white tee 0](https://cloud.githubusercontent.com/assets/907219/10603025/3ed6eb84-771c-11e5-9e04-f7021835db0d.png)
![haxe_tshirt_v01_white tee 1](https://cloud.githubusercontent.com/assets/907219/10603027/3ed75b6e-771c-11e5-91d0-9ab6dfdf0447.png)

**Comment by ConfidantCommunications** [comment]
Nice!

**Comment by waneck** [comment]
@MatthijsKamstra those are all very nice. Thanks for taking the time to work on this.
I'd try to scale down a little bit the ones that have the vertical big logo and see if it looks better, It seems to me that it has not much breathing space.

As for the pixel art, what do you think of something like:
![haxe-test4](https://cloud.githubusercontent.com/assets/178171/10626334/dc61b078-777b-11e5-8915-b9feab624f3d.png)

**Comment by ConfidantCommunications** [comment]
Rotating the pixels looks a bit odd, IMHO.

**Comment by waneck** [comment]
Yeah, I cheated :P . Well, back to the drawing board! Wish our logo was laid out like Zelda's triforce. Like Haxe was the "quadriforce" or something :P

**Comment by ConfidantCommunications** [comment]
If I remember correctly, if you're in Photoshop, you could scale down first to get the pixels, then scale up again by a power of 2 using Nearest Neighbour interpolation.

**Comment by MatthijsKamstra** [comment]
There is a font close to **Helvetica** (which I suspect is used in the original Haxe logo) : http://www.azfonts.net/load_font/hellovetica.html
![](http://www-ru.azfonts.net/fonts_images/samples/005/hellovetica.png)

But I like **Silkscreen**: 
http://www.dafont.com/silkscreen.font
![](http://www.dafont.com/img/preview/s/i/silkscreen1.png)

I can't find a great pixel font that could be used as a bold version, but **04b_19** looks nice: 
http://www.dafont.com/04b-19.font
![](http://www.dafont.com/img/preview/0/4/04b_190.png)

**Comment by markknol** [comment]
I thought it was just Arial Black.

I'm not sure about the pixelated logo. It now looks like a different logo.

**Comment by MatthijsKamstra** [comment]
@markknol that sounds even more plausible. 
![logofont](https://cloud.githubusercontent.com/assets/907219/10660122/e5a5f7e6-78a4-11e5-96a1-9cd2076231b8.png)

If we really, really, really want a pixelated logo we should do it correct and create one by hand. The little experiments I did ended up without the (for lack of a better name) glue flaps...

**Comment by MatthijsKamstra** [comment]
I see no objection about these basic designs, so let's proceed to the next step.

I am guessing nobody want to have a couple boxes full of t-shirts at home and handle shipping.

So....
1. The HaxeFoundation should make a account at  http://www.cafepress.co.uk/ 
2. Decide on "profit" per t-shirt (I would suggest to keep it as low as 2 euro, it should be a conversation starter and not another source of income) 
3. and give me access to the account so I can upload the designs for tshirt and mug
4. add it shop to the website

Who can start this up @Merelleya @ncannasse ????

**Comment by elsassph** [comment]
It's a nice initiative but II don't think a huge Haxe orange logo is a good idea for a T-shirt. Needs something a bit more subtle.

**Comment by bendmorris** [comment]
A cleaner pixel version of the logo: 

![haxe-pixel](https://cloud.githubusercontent.com/assets/544977/10712307/29cf2a8e-7a4c-11e5-8577-debf6992810f.png)

![haxe-pixel-lg](https://cloud.githubusercontent.com/assets/544977/10712315/4c40faf2-7a4c-11e5-8f1a-0f9534129ab9.png)

**Comment by Justinfront** [comment]
Really good quality hoody would be nice, just logo back and front quite large and maybe one on the top front edge of the hoot, no text, also might be nice to have a top with lots of haxe library logos to promote the whole ecosystem and awareness of community, I am quite keen on text less, then people who don't know are more likely to ask since Haxe logo is good without text.  I like the pixel one above.  What is important is to create quality, I was not that impressed with the t-shirts this year.  Quality will be used again and again and the hardcore users won't mind buying the odd quality item and they will be inclinded to wear or use it more if it's good quality.

**Comment by waneck** [comment]
Oh I forgot to comment. @bendmorris I LOVE the pixel version of the logo! Thanks!!

**Comment by MatthijsKamstra** [comment]
I have no idea what the quality is from cafepress, but they are well known. I don't think they will use crap quality.
All I need is an **yes** from the HaxeFoundation!
And I will make hoodies, t-shirts, shorts, pants, oven mitts, and whatever the Haxe community wants.

btw I get that some people like the pixel version of the logo. But I also think it gives a (already) wrong impression of Haxe: that it's build for gaming.

**Comment by Justinfront** [comment]
"I get that some people like the pixel version of the logo. But I also think it gives a (already) wrong impression of Haxe: that it's build for gaming."

on the front using pixel logo.

"Haxe Pixel Logo"
For Games

Then on the back 

"Haxe Logo" 
... And for 
everything else.

**Comment by bendmorris** [comment]
> But I also think it gives a (already) wrong impression of Haxe: that it's build for gaming.

This is a good reason to use multiple designs. Some of us do use it for gaming, and I'd love to have a shirt and mug with no text and a pixelized logo. Tempted to just make one myself on cafepress if the discussion goes on any longer ;-)

**Comment by MatthijsKamstra** [comment]
Wait for it.......
I will keep this subject alive!!!
I am also very interested in a mug and tee myself.

And (@bendmorris ) I agree that there could be more versions so everybody is happy. 
But... (and here comes my "secret" mission) I am trying to attract more JavaScript developers to Haxe and I know no JavaScript game developers. Pixels screams old skool games, and that is not what the majority of js devs do. Although I like the pixel logo, I would never buy it... Because I don't do games. For obvious reasons Haxe has a strong focus on games, but game developers are but a small group of potential new Haxe developers

But perhaps the t-shirt issue is not the best place to discus this. And any "conversation starter" is a plus, so let's do a pixel one as well :P 

@Justinfront that is not a bad idea!

**Comment by MatthijsKamstra** [comment]
Just an update to keep this issue alive
(I still need some sort of okay from @ncannasse  of @Merelleya )

I cleaned the pixel logo a little up (it wasn't symmetrical)
![haxelogopixel_v02](https://cloud.githubusercontent.com/assets/907219/10951582/97d1c096-833e-11e5-85e7-8cdb8742a1d5.png)

![screen shot 2015-11-04 at 21 52 35](https://cloud.githubusercontent.com/assets/907219/10951605/b0a720a2-833e-11e5-89e7-bd2834d13c82.png)

**Comment by manno-xx** [comment]
Since this is still open I guess the t-shirts sold on amazon (http://amzn.com/B016APSIHU) are not the same product discussed here. Does anyone know about the progress of this issue or the origin of the shirts on amazon?

**Comment by MatthijsKamstra** [comment]
I really want to take this a step further, but as you can see it's difficult to get a "GO".
But there are some changes in the HaxeFoundation that I will be waiting for: https://medium.com/@ncannasse/some-words-about-haxe-foundation-e97a4e9d7e41#.if5egdt3c 
(TL;DR : they are looking for a CEO )

About the haxe shirt you found on Amazon: they are from openfl, a test that shouldn't be visible but for some reason is. 
(http://www.amazon.com/s/ref=w_bl_sl_s_ap_web_7141123011?ie=UTF8&node=7141123011&field-brandtextbin=OpenFL)

**Comment by manno-xx** [comment]
Thanks for the reply. I read the post from Nicolas and hope things work out OK soon. I think just for the fun of it I'll order one of the Amazon shirts to see if it comes through. Shipping is set to "in 1 to 2 weeks"...

**Comment by Merelleya** [comment]
Alright so I just spent some time looking at different sites. All of them require the actual assets for the design rather than a picture of a t-shirt with a design.

I am happy to set this up on cafepress if we can agree on 2 or 3 different designs. I think it is great to have some diversity!

Fiene

**Comment by MatthijsKamstra** [comment]
Sure I can give you the files: there in illustrator, but any type I can handle.

I would have a range from the images I provided
- small logo with text (front) (pocket)
- bigger logo with text (front)
- small logo with text (front) and big logo (back)
- bigger logo with text (front) and big logo (back)

and then in two colors : black and white

and I would like to add the mug (black and white)

@Merelleya will you let me know what you need?
(I really didn't do anything ... just placed the logo on a tshirt template)

**Comment by Merelleya** [comment]
apparently just the image filse (psd is fine)

The size is adjustable as long as the resolution is good enough^^

Am 13/01/2016 um 16:57 schrieb Matthijs Kamstra:

**Comment by bablukid** [comment]
just an idea : http://www.unixstickers.com
we could invite those guys at WWX 2016 to sell haxe goodies ( ... and debian, ubuntu, firefox , python stuff ...etc )

**Comment by MatthijsKamstra** [comment]
I am not sure if we are waiting for my input, but if we are:
https://drive.google.com/folderview?id=0B71bEuxrm-kDWEZHTWR4ZlR5QzQ&usp=sharing

I have exports for the mugs and tshirts, more for positioning and dimensions 

To make the prints I have added the files I used (which are not different from the source files from haxe.org).  
Usually vector files are best: `.svg`, `.ai`, `.eps` because they scale to infinity (and beyond).
Just in case, I've added png but I have no idea how big they should be. 
Will you let me know when they are too small?

**Comment by mikicho** [comment]
@MatthijsKamstra Looks great! it's possible to creae a left handed version of the mug? (the logo on the other side..)

**Comment by MatthijsKamstra** [comment]
I was reading through all these issues, and thought it would be a great idea to have a haxe-tshirt/mug. Still work in progress?

**Comment by Merelleya** [comment]
We made it!
